_docs/crowfacts.md

```diff
@@ -19,13 +19,41 @@ More information coming soon!
 More information coming soon!
 
 ## Architecture
-
-More information coming soon!
+To create a functional user website, which acheived the above goals, we used AWS services to host the website and its services on the cloud. The websites design is described below.
+
+- An [S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html) was used as the website host, and was configured with a custom domain, as well as [SSL certification for HTTPS traffic](https://www.freecodecamp.org/news/simple-site-hosting-with-amazon-s3-and-https-5e78017f482a/).
+- This website was designed to give users access to facts, or to make their own.
+- These user actions are made via [API gateway calls to Lambda functions](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-getting-started-with-rest-apis.html), which execute GET or PUT calls.
+- Functions calls create a link to either the CrowFacts or FunFacts [DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Introduction.html) tables.
+    - **CrowFacts**: NoSQL database that returns species information on crows, found by combination of partition and sort key.
+        - Primary key: "CrowSpecies" + "habitat"
+        - Sort: 
+            - "description": a short decription of the crows appearance
+            - "SubSpecies": specifies what subspecies a crow is, e.g., Florida Crow 
+            - "scientific": the scientific name of the crow species 
+            - "image": url to an image of the crow species
+    - **FunFacts**: A mixture of actual fun facts about crows, and user inputted facts, sanitized at time of Lambda call.
+        - Primary key: "fact"
+        - Sort: 
+            - "source": the source of who inputted the fact
+- Success of GET or PUT call is logged via [AWS Cloudwatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html).
+- If successful, the information will return succesful to API Gateway, which will return the information retrieved.
+- Based on if user made a call to retrieve crow information, or fun facts, the S3 bucket will create a visual representation of the information
+    - Website uses the image url's in the "image" column to show crow photos. 
+    - Additionally, the website uses the "description" column to give a verbal description of the crows.
+
+[Visit the website here.](https://crowfacts.uwbhacks.com/)
+
+[View the example Lambda PUT call here.](https://github.com/UWB-ACM/crowfacts/blob/master/lambda_put_user_fact/lambda_function.py)
+
+[View the example Lambda GET call here.](https://github.com/UWB-ACM/crowfacts/blob/master/lambda_get_user_facts/lambda_function.py)
+
+
+The diagram of the system flow is below:
```

Comment:
```suggestion
The diagram of the system flow is available at:
```

Reply:
I am fine with the original wording. 

As a side note, I like putting images into the repository under the `/assets` folder -- this lets us avoid link rot.

Reply:
Kept the original wording, and updated the image link to be hosted on the repo   o7